From: Chuck Ebbert <cebbert@redhat.com>

CIFS: Fix DNS resolver build

In file included from fs/cifs/dns_resolve.c:29:
fs/cifs/dns_resolve.h:27: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'cifs_init_dns_resolver'

Just remove the __init and __exit attributes from the init and exit
functions. __exit was removed upstream in 51c20fcced5badee0e2021c6c89f44aa3cbd72aa
anyway, and there's no point trying to save every byte by fixing
this properly.

Signed-Off-By: Chuck Ebbert <cebbert@redhat.com>

--- a/fs/cifs/dns_resolve.c
+++ b/fs/cifs/dns_resolve.c
@@ -176,7 +176,7 @@ out:
 	return rc;
 }
 
-int __init cifs_init_dns_resolver(void)
+int cifs_init_dns_resolver(void)
 {
 	struct cred *cred;
 	struct key *keyring;
@@ -226,7 +226,7 @@ failed_put_cred:
 	return ret;
 }
 
-void __exit cifs_exit_dns_resolver(void)
+void cifs_exit_dns_resolver(void)
 {
 	key_revoke(dns_resolver_cache->thread_keyring);
 	unregister_key_type(&key_type_dns_resolver);
--- a/fs/cifs/dns_resolve.h
+++ b/fs/cifs/dns_resolve.h
@@ -24,8 +24,8 @@
 #define _DNS_RESOLVE_H
 
 #ifdef __KERNEL__
-extern int __init cifs_init_dns_resolver(void);
-extern void __exit cifs_exit_dns_resolver(void);
+extern int cifs_init_dns_resolver(void);
+extern void cifs_exit_dns_resolver(void);
 extern int dns_resolve_server_name_to_ip(const char *unc, char **ip_addr);
 #endif /* KERNEL */
 
